integration-tests/cypress/integration/api/devices/device_register.ts

```diff
@@ -0,0 +1,78 @@
+/// <reference path="../../../support/index.d.ts" />
+
+import { getTestName } from "../../../commands/names";
+import { getCreds } from "../../../commands/server";
+
+describe("Device register", () => {
+  const camsGroup = "cams";
+  const otherCams = "other cams";
+
+  const KEEP_DEVICE_NAME = false;
```

Comment:
why are these constants? and not just set when calling the function

Reply:
inherited this approach - its throughout the existing test code so continued for consistency.